integreat_cms/release_notes/current/unreleased/2699.yml

```diff
@@ -0,0 +1,2 @@
+en: fixed the max-width of the feedback object column for admin feedback and region feedback
+de: 
```

Comment:
Could you please add a German translation for the release note? :blush:

Reply:
Sure :)